packages/venia-ui/lib/components/CheckoutPage/checkoutPage.js

```diff
@@ -0,0 +1,190 @@
+import React, { useCallback, Fragment } from 'react';
+
+import { useCheckoutPage } from '@magento/peregrine/lib/talons/CheckoutPage/useCheckoutPage';
+
+import { Title } from '../../components/Head';
+import Button from '../Button';
+import PriceSummary from './PriceSummary';
+import PriceAdjustments from './PriceAdjustments';
+import PaymentInformation from './PaymentInformation';
+import ShippingMethod from './ShippingMethod';
+import ShippingInformation from './ShippingInformation';
+
+import { renderIf } from './utilities';
+import { mergeClasses } from '../../classify';
+
+import defaultClasses from './checkoutPage.css';
+
+const GuestCheckoutOptions = props => {
+    const { classes, handleSignIn } = props;
+
+    return (
+        <div className={classes.signin_container}>
+            <div className={classes.heading_container}>
+                <h1 className={classes.heading}>Quick Checkout</h1>
+            </div>
+            <Button
+                className={classes.sign_in}
+                onClick={handleSignIn}
+                priority="high"
+            >
+                {'Login to Checkout Faster'}
```

Comment:
In the mocks this is a button that says "Sign In". Did you get UX feedback that differs from the mocks for this?

Reply:
Yeah, the mocks are old. @soumya-ashok showed me the latest mocks which are not updated in the ticket yet.